lua/epics.lua

```diff
@@ -111,6 +111,25 @@ function M.setup(opts)
 
 	M.options = vim.tbl_extend("force", M.options, opts)
 
+	vim.filetype.add {
+		extension = {
+			ioc = "make",
+			Db = "make",
+		},
+		filename = {
+			["RULES.Db"] = "make",
+			["RULES.ioc"] = "make",
+			["CONFIG_STIE"] = "make",
+			["CONFIG_STIE.local"] = "make",
+			["CONFIG_STIE_ENV"] = "make",
+		},
+		pattern = {
+			[".*/configure/os/CONFIG.*"] = "make",
+			[".*/configure/CONFIG.*"] = "make",
+			[".*/configure/RULES.*"] = "make",
+			[".*/configure/RELEASE"] = "make",
```

Comment:
Some suggestions to match more files:

```suggestion
			[".*/configure/os/CONFIG.*"] = "make",
			[".*/configure/CONFIG.*"] = "make",
			[".*/configure/RULES.*"] = "make",
			[".*/configure/RELEASE.*"] = "make",
			[".*/cfg/CONFIG.*"] = "make",
			[".*/cfg/DIR_RULES.*"] = "make",
			[".*/cfg/RULES.*"] = "make",
			[".*/cfg/TOOLCHAIN.*"] = "make",
			[".*/cfg/TOP_RULES.*"] = "make",
```